### PR TITLE
Enable deeply nested message validation options

### PR DESCRIPTION
**Pull request by Calebjh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+FROM golang:1.13.0-alpine AS builder
 
 LABEL stage=server-intermediate
 
 WORKDIR /go/src/github.com/infobloxopen/protoc-gen-atlas-query-validate
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /out/usr/bin/protoc-gen-atlas-query-validate main.go
 
-FROM infoblox/atlas-gentool:latest AS runner
+FROM infoblox/atlas-gentool:v21.3 AS runner
```
Comment (Evgeniy-L) [inline Dockerfile]: As far as I remember the idea behind this was to be able to create gentool with current version of protoc-gen-atlas-query-validate for testing. Pointing to a specific version here may not be good for future.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### Purpose
 
-A [protobuf](https://developers.google.com/protocol-buffers/) compiler plugin 
+A [protobuf](https://developers.google.com/protocol-buffers/) compiler plugin
```
Comment (Evgeniy-L) [inline README.md:5]: Is this space removal thing something that IDE automatically does?

**Review by Evgeniy-L — COMMENTED** [review]
Just a small note regarding Dockerfile below. Do not see any concerns regarding logic itself. May be better if @askurydzin review the logic as he wrote it in the first place.

**Review by askurydzin — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (Evgeniy-L) [reply Dockerfile]: suggest to revert it back to `latest`

**Review by Evgeniy-L — APPROVED** [review]
Approving, but please note the comment about gentool image version

File: README.md (hunk 2 above)
Comment (Calebjh) [reply README.md:5]: Yes, it's part of the formatting executed whenever I save a file.

**Pull request merged**